core/ui/DownloadFullWiki.tid

```diff
@@ -0,0 +1,17 @@
+title: $:/core/ui/DownloadFullWiki
+
+\whitespace trim
+\rules except wikilink
+
+To download the standard single-file version of your wiki:
+
+<$wikify name="site-title" text={{$:/config/SaveWikiButton/Filename}}>
+<$let publishFilter="""-[[$:/config/SaveWikiButton/Template]] -[[$:/plugins/tiddlywiki/filesystem]] -[[$:/plugins/tiddlywiki/tiddlyweb]]""">
+<$button tooltip="Download fully standalone wiki" aria-label="download full wiki" class="tc-btn-big-green">
+<$action-sendmessage $message="tm-download-file" $param="$:/core/save/all" publishFilter=<<publishFilter>> filename=<<site-title>>/>
+{{$:/core/images/download-button}}
+&#32;
```

Comment:
please do not user whitespace to change the layout. Use `<span class="tc-tiny-gap-left">` instead. It will allow us to modify gaps globally in the future if we need to. See: utility classes can be found here: https://github.com/pmario/TiddlyWiki5/blob/7a7d83784c91eaa3c385918918491c1416c31fba/themes/tiddlywiki/vanilla/base.tid#L3046-L3091

Reply:
Thanks @pmario. Done